lang/cs/Org.Apache.REEF.Experimental.Tests/TestParquetReader.cs

```diff
@@ -0,0 +1,109 @@
+// under the License.
+
+using System.IO;
+using System.Runtime.Serialization;
+using Org.Apache.REEF.Experimental.ParquetReader;
+using Org.Apache.REEF.Tang.Implementations.Tang;
+using Org.Apache.REEF.Tang.Interface;
+using Org.Apache.REEF.Tang.Util;
+using Xunit;
+using System.Diagnostics;
+
+namespace Org.Apache.REEF.Experimental.Tests
+{
+    public sealed class TestParquetReader
+    {
+        [DataContract(Name = "User", Namespace = "parquetreader")]
+        internal class User
+        {
+            [DataMember(Name = "name")]
+            public string name { get; set; }
+
+            [DataMember(Name = "age")]
+            public int age { get; set; }
+
+            [DataMember(Name = "favorite_color")]
+            public string favorite_color { get; set; }
+        }
+
+        private sealed class WorkingDir
+        {
+            public DirectoryInfo dir = new DirectoryInfo(Directory.GetCurrentDirectory());
+            public WorkingDir()
+            {
+                while (!dir.FullName.EndsWith(@"reef\lang"))
+                {
+                    dir = Directory.GetParent(dir.FullName);
+                }
+            }
+        }
+
+        DirectoryInfo dir = new WorkingDir().dir;
+
+        [Fact]
+        public void TestEnvironment()
+        {
+            // Make sure test data file.parquet exist
+            Assert.True(File.Exists(dir.FullName + @"\java\reef-experimental\src\test\resources\file.parquet"));
+
+            // Make sure test data jar file exist
+            Assert.True(File.Exists(Directory.GetFiles(dir.FullName + @"\java\reef-experimental\target\", "*jar-with-dependencies.jar")[0]));
+
+            // Make sure all dependencies are in the target\dependency folder
+            Assert.True(Directory.Exists(dir.FullName + @"\java\reef-experimental\target\dependency\"));
+        }
+
+        [Fact]
+        public void TestReadParquetFile()
+        {
+            // Identify parquet file and class path
+            var parquetPath = dir.FullName + @"\java\reef-experimental\src\test\resources\file.parquet";
+            var classPath =
+                dir.FullName + @"\java\reef-experimental\target\dependency\*;" +
+                Directory.GetFiles(dir.FullName + @"\java\reef-experimental\target\", "*jar-with-dependencies.jar")[0];
+
+            Assert.True(File.Exists(parquetPath));
+
+            Debug.WriteLine("Parquet Path: {0}", parquetPath);
+            Debug.WriteLine("Jar Path: {0}", classPath);
+
+            ITang tang = TangFactory.GetTang();
+            IConfiguration conf = tang.NewConfigurationBuilder()
+              .BindNamedParameter<ParquetPathString, string>(GenericType<ParquetPathString>.Class, parquetPath)
+              .BindNamedParameter<JarPathString, string>(GenericType<JarPathString>.Class, classPath)
+              .Build();
+            IInjector injector = tang.NewInjector(conf);
+
+            var reader = injector.GetInstance<ParquetReader.ParquetReader>();
```

Comment:
Use a `using` construct here. Otherwise, `.Dispose()` isn't guaranteed to be called.